src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
+@import "./variables.scss";
+
+@import "./header.scss";
+@import "./product-card.scss";
+@import "./stars.scss";
+
+* {
```

Comment:
do not use `*` to reset margins, it will reset margins for all elements, it's not good for browsers and performance